server/views/pages/induction/checkYourAnswers/partials/_educationAndTraining.njk

```diff
@@ -0,0 +1,64 @@
+  </h3>
+
+  <table class="govuk-table govuk-!-margin-bottom-6" data-qa="qualifications">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">Level</th>
+        <th scope="col" class="govuk-table__header">Subject</th>
+        <th scope="col" class="govuk-table__header">Grade</th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      {% for item in inductionDto.previousQualifications.qualifications %}
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell" data-qa="qualifications-level-{{ loop.index }}">{{ item.level | formatQualificationLevel }}</td>
+          <td class="govuk-table__cell" data-qa="qualifications-subject-{{ loop.index }}">{{ item.subject }}</td>
+          <td class="govuk-table__cell" data-qa="qualifications-grade-{{ loop.index }}">{{ item.grade }}</td>
+        </tr>
+      {% endfor %}
+    </tbody>
+  </table>
+
+{% endif %}
+
+<h3 class="govuk-heading-s govuk-!-margin-top-2 govuk-!-margin-bottom-3">Other qualifications and history</h3>
+
+<dl class="govuk-summary-list govuk-!-margin-bottom-8">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key govuk-!-width-one-quarter ">
+      Highest level of education before prison
+    </dt>
+    <dd class="govuk-summary-list__value" data-qa="educationLevel">
+      {{ inductionDto.previousQualifications.educationLevel | formatEducationLevel if inductionDto.previousQualifications.educationLevel }}
+    </dd>
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link govuk-link--no-visited-state" href="/prisoners/{{ prisonerSummary.prisonNumber }}/induction/highest-level-of-education" data-qa="educationLevelLink">
+        Change<span class="govuk-visually-hidden"> highest level of education before prison</span>
+      </a>
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key govuk-!-width-one-quarter">
+      Other training or qualifications
+    </dt>
+    <dd class="govuk-summary-list__value" data-qa="additionalTraining">
+      {% for item in inductionDto.previousTraining.trainingTypes %}
+        <div data-qa="additionalTraining-{{ loop.index }}">{{ item | formatAdditionalTraining }}{{ ' - ' + inductionDto.previousTraining.trainingTypeOther if item === 'OTHER' }}{{ ',' if inductionDto.previousTraining.trainingTypes.length !== loop.index }}</div>
+      {% endfor %}
+    </dd>
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link govuk-link--no-visited-state" href="/prisoners/{{ prisonerSummary.prisonNumber }}/induction/additional-training" data-qa="additionalTrainingLink">
+        Change<span class="govuk-visually-hidden"> other training or qualifications they have</span>
+      </a>
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row">
```

Comment:
Weird empty row!
Given that the scope of what we are doing is to take the views and functionality verbatim from the CIAG UI, then to that end that is what we have done here
I'm guessing it was added to add spacing perhaps? But I'm not going to change it as part of these stories.